File: selectionsort.c
#include<stdio.h>

int main()
{
    int a[]={3,2,11,8,6,5};
    int i,j,min,temp;
    for(i=0;i<5;i++)
    {
        min=i;
        for(j=i+1;j<6;j++)
        {
            if(a[j]<a[min])
            {
                min=j;
            }
        }
        temp=a[min];
        a[min]=a[i];
        a[i]=temp;
    }
    for(i=0;i<6;i++)
    {
        printf("%d",a[i]);
    }
}

File: selectionsort.py
#program using selection sort
a=[2,4,7,11,8,6]
for i in range(0,len(a)-1):
    min=i
    for j in range(i+1,len(a)):
        if(a[j]<a[min]):
            min=j

    temp=a[min]
    a[min]=a[i]
    a[i]=temp

print(a)

